src/fparser/common/readfortran.py

```diff
@@ -1100,6 +1101,25 @@ def warning(self, message, item=None):
 
     # Auxiliary methods for processing raw source lines:
 
+    @staticmethod
+    def replace_omp_sentinels(line, regex):
+        """Checks if the specified line matches the regex, which represents
+        a conditional OpenMP sentinel. If it is a match, the sentinel (which
+        must be the first group in the regex) is replaced with two spaces.
+        :param line: the line to check if it contains an OpenMP sentinel
```

Comment:
Thanks for adding this utility. Please could you put a blank line before the first ":param ..." and you can just include the type on the same line:

    :param str line: the line to check....

Reply:
Done.